File: CS/MessengerExample/Model/Employee.cs
using DevExpress.Mvvm;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MessengerExample.Model {
    public class Employee : BindableBase {
        [ReadOnly(true)]
        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public void Refresh() {
            RaisePropertyChanged();
        }

        public Employee() {
        }
        public Employee(string firstName, string lastName) {
            FirstName = firstName;
            LastName = lastName;
        }
    }
}

File: VB/MessengerExample/Model/Employee.vb
Imports DevExpress.Mvvm
Imports System.ComponentModel
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

Namespace MessengerExample.Model
    Public Class Employee
        Inherits BindableBase

        <[ReadOnly](True), Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)> _
        Public Property ID() As Integer
        Public Property FirstName() As String
        Public Property LastName() As String
        Public Sub Refresh()
            RaisePropertyChanged()
        End Sub

        Public Sub New()
        End Sub
        Public Sub New(ByVal firstName As String, ByVal lastName As String)
            Me.FirstName = firstName
            Me.LastName = lastName
        End Sub
    End Class
End Namespace